### PR TITLE
13097 improve VM tunables for modern systems

### PR DESCRIPTION
**Pull request by citrus-it**
13107 pageout should use hrtime instead of lbolt

### CODE REVIEW

**Review by oetiker — APPROVED** [review]
:+1:

File: usr/src/uts/common/os/vm_pageout.c [hunk 1]
```diff
@@ -1024,29 +1207,40 @@ pageout_scanner(void *a)
...
+	 * the loop:
 	 */
-	CPU_STATS_ADDQ(CPU, vm, pgrrun, 1);
-	count = 0;
+	laps = 0;
 
-	/* Kernel probe */
-	TNF_PROBE_2(pageout_scan_start, "vm pagedaemon", /* CSTYLED */,
-	    tnf_ulong, pages_free, freemem, tnf_ulong, pages_needed, needfree);
+	DTRACE_PROBE(pageout__start);
```
Comment (danmcd) [inline usr/src/uts/common/os/vm_pageout.c:1218]: Lose this one, because there's a more-data version of it on current-line 1241.

**Comment by citrus-it** [comment]
Tested with full build and onu. Clean mail_msg.
Further testing to be continued once it's merged into bloody.

**Comment by danmcd** [comment]
Don't forget that @papertigers (add him to this PR?) is also testing this on SmartOS.

**Comment by papertigers** [comment]
Yep, OmniOS box is running these bits. Spinning up a few test SmartOS VMs with various memory profiles under bhyve. Will report back shortly (or after a few meetings depending on how quickly I get things setup).

**Review by hadfl — APPROVED** [review]

**Pull request merged**